src/Geometry/localgeometry.jl

```diff
@@ -55,7 +79,9 @@ struct SurfaceGeometry{FT, N}
     normal::N
 end
 
-undertype(::Type{LocalGeometry{I, C, FT, S}}) where {I, C, FT, S} = FT
+undertype(
+    ::Type{LocalGeometry{I, C, FT, ∂x∂ξT, ∂ξ∂xT, gⁱʲT, gᵢⱼT}},
+) where {I, C, FT, ∂x∂ξT, ∂ξ∂xT, gⁱʲT, gᵢⱼT} = FT
```

Comment:
```suggestion
undertype(::Type{LocalGeometry{I, C, FT}}) where {I, C, FT} = FT
```
The new type parameters aren't used here.

Reply:
I think all type parameters are needed, unless we use `<:`

Reply:
I'll just use the `FullLocalGeometry` alias